defaultconfigs/immersiveengineering-server.toml

```diff
@@ -352,7 +352,7 @@
 		air_exposure = 0.0
 		#The maximum size of a vein. Set to 0 to disable generation
 		#Range: > 0
-		vein_size = 6
+		vein_size = 0

```

Comment:
watch out for configswapper too. You'll want to update those as well for anything including ore gen, like IE.